src/xdocs/config_javadoc.xml

```diff
@@ -860,38 +862,54 @@ public int checkReturnTag(final int aTagIndex,
 
       <subsection name="Examples" id="JavadocParagraph_Examples">
         <p>
-          Default configuration:
+          To configure the default check:
         </p>
         <source>
 &lt;module name=&quot;JavadocParagraph&quot;/&gt;
+        </source>
+        <p>By default, the check will report a violation if there is a new line
+          or whitespace after the &lt;p&gt; tag: </p>
+        <source>
+&#47;**
+ * No tag (ok).
+ *
+ * &lt;p&gt;Tag immediately before the text (ok).
+ *
+ * &lt;p&gt;
+ * New line after tag (violation).
+ *
+ * &lt;p&gt; Whitespace after tag (violation).
+ *
```

Comment:
please do us favor and add extra line to show that violation appear if no empty line between paragraphs.
`* &lt;p&gt; no empty line before p tag (violation).`

and the same line to the second example.

Reply:
done